sister/word_embedders.py

```diff
@@ -12,7 +12,7 @@ def get_fasttext(lang: str = "en"):
     # Download.
     urls = {
             "en": "https://dl.fbaipublicfiles.com/fasttext/vectors-wiki/wiki.simple.zip",
-            "ja": "https://dl.fbaipublicfiles.com/fasttext/vectors-wiki/wiki.ja.zip",
```

Comment:
could you not erase "ja" (stands for Japanese)?
Instead, you can simply add "hi" as new line.